### PR TITLE
ci: Migrate CI to Bitrise (SDKCF-4710)

### PR DESCRIPTION
**Pull request by maureenorea-clores**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [![Maven Central](https://img.shields.io/maven-central/v/io.github.rakutentech.inappmessaging/inappmessaging)](https://search.maven.org/artifact/io.github.rakutentech.inappmessaging/inappmessaging)
-[![CircleCI](https://circleci.com/gh/rakutentech/android-inappmessaging.svg?style=svg)](https://circleci.com/gh/rakutentech/android-inappmessaging)
+![Build status](https://app.bitrise.io/app/e9cc83da00ffd2b1/status.svg?token=4E3R-Baoxp0iPav7uiw1sA&branch=master)
```
Comment (leila-gu) [inline README.md:2]: Is it safe to expose the account access token here? I wonder if GitHub encrypted secrets can be used in .md files?

Comment (maureenorea-clores) [reply README.md:2]: I think this token is only used to access the build status image and cannot be used for other purpose.
@donnie-jp Please advise if it is still ok to use this badge image in README.
Similarly, it is added in https://raw.githubusercontent.com/rakutentech/ios-inappmessaging/master/README.md as well.

Comment (donnie-jp) [reply README.md:2]: Yeah it seems like it's fine.
When you click the badge on a bitrise project it says the token is a "status image API token". It isn't a general purpose token. Also, on [here ](https://blog.bitrise.io/post/announcing-status-images) it says
> we created a new read-only token just for interacting with the status image, so you can show it anywhere and still have your project in full safety

File: bitrise.yml [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+            #!/bin/bash
+            set -e
+            ./gradlew generatePublishableDocs
+            if [[ ! $BITRISE_GIT_TAG == *"-"* ]]; then
+              git checkout gh-pages
+              cp -R inappmessaging/build/publishableDocs/docs/. ./docs
+              cp -R inappmessaging/build/publishableDocs/_versions/. ./_versions
+              git add docs _versions
+              git config user.name "CI Publisher"
+              git config user.email "dev-opensource@mail.rakuten.com"
```
[the remark this answers is not in the document]
Comment (maureenorea-clores) [reply bitrise.yml]: Removed.
The default "please-set-your-email@bitrise.io" will be displayed in commit logs instead.
https://devcenter.bitrise.io/en/builds/configuring-build-settings/setting-your-git-credentials-on-build-machines.html

**Review by leila-gu — APPROVED** [review]
LGTM

File: bitrise.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (donnie-jp) [reply bitrise.yml]: Sorry - it seems to be a public email. At least it's used on https://rakutentech.github.io/ as the contact email!
So it's fine to restore it.

[the remark this answers is not in the document]
Comment (maureenorea-clores) [reply bitrise.yml:151]: Thanks for checking! I restored it

**Review by imaginaris — APPROVED** [review]

**Review by vincentdelmundo — APPROVED** [review]
LGTM

**Pull request merged**